tests/kernel/smp/src/main.c

```diff
@@ -372,7 +372,7 @@ ZTEST(smp, test_coop_resched_threads)
 	 * since we don't give up current CPU, last thread
 	 * will not get scheduled
 	 */
-	spawn_threads(K_PRIO_COOP(10), num_threads, !EQUAL_PRIORITY,
+	spawn_threads(K_PRIO_COOP(MAX_NUM_THREADS), num_threads, !EQUAL_PRIORITY,
```

Comment:
how does this relate to ACPICA? (as above in the smp's prj.conf)

Make it its own commit, and have a proper commit message explaining the change.